components/docs-chef-io/content/habitat/plan_variables.md

```diff
@@ -17,31 +17,47 @@ pkg_prefix
 : The absolute path for your package.
 
 pkg_dirname
-: Set to `{pkg_name}-{pkg_version}` by default. If a .tar file extracts to a directory that's different from the filename, then you would need to override this value to match the directory name created during extraction.
+: The directory created when Habitat extracts an archive.
+
+  If a .tar file extracts to a directory that's different from the filename, then you would need to override this value to match the directory name created during extraction.
+
+  Default value: `{pkg_name}-{pkg_version}`
```

Comment:
To have some consistency, let's say the default value is `${pkg_name}-${pkg_version}`, which is the standard Bash syntax for referencing variables.

Reply:
Also, this is the only variable of the ones that this PR touches that could legitimately be changed by a user; everything else should be treated as a read-only variable.